lib/broccoli/ember-app.js

```diff
@@ -874,7 +875,7 @@ EmberApp.prototype.import = function(asset, options) {
     return;
   }
 
```

Comment:
as node internally normalizes most `\`  or `/` to platform specific separator, we should do the same

```js
assetPath = assetPath.replace(path.sep, '/')
```
before that conditional, i believe will do the trick